### DISCUSSION TITLE
阿里云盘突然不能用了

### DISCUSSION THREAD

**Discussion by weidongzhou1994** [AlistGo/alist#4825] [Q&A]
<img width="908" alt="image" src="https://github.com/alist-org/alist/assets/40510496/8cea5b4f-20d5-4abc-bf87-f1c147970da4">

**Comment by Whale778** [comment]
Me too!

**Comment by QiuYeDx** [comment]
我也是一样，上午还好好的，刚才突然就不行了，赶紧来仓库看看

**Reply by QiuYeDx** [reply]
<img width="967" alt="截屏2023-07-22 14 56 01" src="https://github.com/alist-org/alist/assets/71062076/7fd1b081-a0f0-4d6f-a80e-1b72f56d0b56">

**Reply by QiuYeDx** [reply]
换了一下令牌链接，保存一下，发现刷新令牌失败，换回原来的也不行了。
<img width="478" alt="截屏2023-07-22 15 05 15" src="https://github.com/alist-org/alist/assets/71062076/5e02a739-2ec8-4ae8-ac52-a77d980dbfb1">

**Reply by QiuYeDx** [reply]
我的版本是3.17.0，难道是需要更新版本吗？

**Comment by anwen-anyi** [comment]
阿里云盘网络波动，现在已经恢复正常了。

**Reply by QiuYeDx** [reply]
确实恢复了，感谢提醒！
